.buildkite/scripts/steps/checks/bazel_packages.sh

```diff
@@ -6,5 +6,8 @@ source .buildkite/scripts/common/util.sh
 
 echo --- Check Bazel Packages Manifest
 node scripts/generate packages_build_manifest
-
 check_for_changed_files 'node scripts/generate packages_build_manifest' true
+
+echo --- Check Codeowners Manifest
```

Comment:
Can we skip this check if it's not `main`?

Reply:
Oh, you mean so that we don't have to disable this when we fork new branches? Good call, I'll only run this if the `.github/CODEOWNERS` file already exists.